document-picture-in-picture/main.js

```diff
@@ -0,0 +1,90 @@
+
+const videoPlayer = document.getElementById("player");
+const playerContainer = document.getElementById("container");
+let togglePipButton;
+let pipActive = false;
```

Comment:
As shared earlier, I would remove this extra variable and simply use `window.documentPictureInPicture.window` instead.

Reply:
Good call, this is a lot simpler. I'll remove it from here in my next commit, and then update the docs as appropriate.